src/foundry/client-esm/canvas/regions/shape.d.mts

```diff
@@ -54,48 +62,57 @@ declare abstract class RegionShape<ShapeData extends BaseShapeData> {
    * @internal
    */
   protected abstract _drawShape(graphics: PIXI.Graphics): void;
-}
-
-// TODO(Eon): Is this even necessary? It's unexported in core.
 
-/** A circle of a {@link Region} */
-declare class RegionCircle extends RegionShape<CircleShapeData> {
-  
-  constructor(data: CircleShapeData);
+  #regionShape: true;
+}
 
-  /** The vertex density epsilon used to create a polygon approximation of the circle. */
-  static #VERTEX_DENSITY_EPSILON: number;
+/**
+ * A circle of a {@link Region}
+ * @privateRemarks Technically this should extend CircleShapeData, but it's easier to instantiate if it's RegionShape.ShapeData.
```

Comment:
nit; missing \`s for code formatting.

Secondly I'd write "Technically `ShapeData` should extend `CircleShapeData`" because when I initially read this I thought you meant `RegionCircle` should.